modify.md

```diff
@@ -54,7 +54,27 @@ class $modify(MenuLayer) {
 
 (or you could be questionable and do `this->MenuLayer::init()` but who am I to judge)
 
-This pattern is quite common inside cocos2d, and therefore in GD too. Here, we modify the `MenuLayer::init` so that it will create our label inside the layer after running the original function. If `MenuLayer::init` returns false something has gone horribly wrong and GD will probably crash but it is important to stick to the idioms for code consistency. 
+This pattern is quite common inside cocos2d, and therefore in GD too. Here, we modify the `MenuLayer::init` so that it will create our label inside the layer after running the original function. If `MenuLayer::init` returns false something has gone horribly wrong and GD will probably crash but it is important to stick to the idioms for code consistency.
+
+We can also improve this function by getting GD Window Size that can later be used to place stuff more precisely.
+
+```cpp
+class $modify(MenuLayer) {
+    bool init() {
+        if (!MenuLayer::init()) return false;
+
+        auto winSize = CCDirector::sharedDirector()->getWinSize();
+
+        auto label = CCLabelBMFont::create("Hello world!", "bigFont.fnt");
+        label->setPosition(winSize.width / 2, winSize.height / 1.35);
+        this->addChild(label);
+
+        return true;
+    }
+};
+```
+
+Here we use `CCDirector::sharedDirector()->getWinSize();` to get the GD window size and assign it to `auto winSize` then we modify the `label->setPosition` setting `winSize.width / 2` (center) and `winSize.height / 1.35` which is right under the main Geometry Dash text/logo in main menu.
```

Comment:
* Remove the semicolon from the end of `CCDirector::sharedDirector()->getWinSize();` (inline code blocks shouldn't contain semicolons)
 * Change `auto winSize` to `winSize` (datatype doesn't need to be rementioned alongside the identifier)
 * We don't 'modify' `label->setPosition`, we 'call' it.
 * Doing `winSize.height / 1.35` to position a node under the main logo is not exactly the greatest plan. In general, if you are positioning something and you're dividing a variable by a factor that isn't a power of 2, you're probably doing something wrong. It would be better as `winSize.height / 2 + 40.f` or however many pixels you need to move the label up to be under the title (positioning under the title in any case should really be done by first getting the title node and positioning relative to it)

Reply:
Removing semicolon from `CCDirector::sharedDirector()->getWinSize();` will result in `syntax error: 'auto' should be preceded by ';'`

Trying to change `auto winSize` to `winSize` will result in `'winSize': undeclared identifier`